operator/main.go

```diff
@@ -343,10 +343,16 @@ func main() {
 		Tracer:   otel.Tracer("keptn/operator/evaluation"),
 		Meters:   meters,
 	}
+
 	if err = (evaluationReconciler).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "KeptnEvaluation")
 		os.Exit(1)
 	}
+
+	if err = (&lifecyclev1alpha2.KeptnApp{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "KeptnApp")
```

Comment:
```suggestion
		setupLog.Error(err, "unable to create conversion webhook", "webhook", "KeptnApp")
```

Reply:
This will actually set up all the webhooks :)
Just we only have a conversion webhook for now so it will only set up that, but as soon as we have validation or defaulting webhooks, those would be set up as well.